src/main/kotlin/org/neo4j/graphql/CrudOperations.kt

```diff
@@ -34,6 +35,35 @@ fun augmentedSchema(ctx: Translator.Context, type: ObjectTypeDefinition): Augmen
     } else result
 }
 
+fun augmentedSchema(ctx: Translator.Context, source: ObjectTypeDefinition, target: ObjectTypeDefinition): Augmentation? {
+    val sourceIdField = source.fieldDefinitions.find { it.type.name() == "ID" }
```

Comment:
we should probably add an extension method to OTD for this

Reply:
ok